src/scripts/main.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+  const wallRect = wall.getBoundingClientRect();
+
+  if (!e.target.closest('.wall')) {
+    return;
+  }
+
+  let coordX =
+    e.clientX - wall.clientLeft - wallRect.left - spider.clientWidth / 2;
+  let coordY =
```

Comment:
Similar to `coordX`, a comment explaining the calculation of `coordY` would improve code readability.